xml/deployment_scenarios.xml

```diff
@@ -0,0 +1,1047 @@
+   <para>
+    You will provide replacements for these resources on a dedicated server
+    inside your internal (airgapped) network.
+   </para>
+   <para>
+    The internal mirror must be updated with data retrieved from the original
+    upstream sources; in a trusted and secure fashion. To achieve this, you will
+    need an additional mirroring server outside of the airgapped network which
+    acts as a first stage mirror and allows retrieving data from the internet.
+   </para>
+   <para>
+    Updating of mirrors happens in three stages.
+   </para>
+   <orderedlist>
+    <listitem>
+     <para>
+      Update the external mirror from upstream.
+     </para>
+    </listitem>
+    <listitem>
+     <para>
+      Transfer the updated data onto a trusted storage device.
+     </para>
+    </listitem>
+    <listitem>
+     <para>
+      Update the internal mirror from the trusted storage device.
+     </para>
+    </listitem>
+   </orderedlist>
+   <para>
+    Once the replacement sources are in place, the key components are
+    reconfigured to use the mirrors as their main sources.
+   </para>
+  </sect3>
+
+  <sect3>
+   <title>RPM Package Repository Mirroring</title>
+   <para>
+    Mirroring of the RPM repositories is handled by the
+    <link xlink:href="https://www.suse.com/documentation/sles-15/book_rmt/data/book_rmt.html">&rmtool;</link>
+    for &sls; 15. The tool provides functionality that mirrors the upstream
+    &suse; package repositories on the local network. This is intended to
+    minimize reliance on &suse; infrastructure for updating large volumes of
+    machines. The airgapped deployment uses the same technology to provide the
+    packages locally for the airgapped environment.
+   </para>
+   <para>
+    &sls; bundles software packages in so called modules. You must enable all
+    the default modules for &sle; and in addition the <literal>Containers</literal>
+    module. All enabled modules need to be mirrored inside the airgapped network
+    in order to provide the necessary software for other parts of this scenario.
+   </para>
+   <para>
+    RMT will provide a repository server that holds the packages
+    and related metadata for &mos;; to install them like from the upstream
+    repository. Data is synchronized once a day to the external mirror
+    automatically or can be forced via the CLI.
+   </para>
+   <para>
+    You can copy this data to your trusted storage at any point
+    and update the internal mirror.
+   </para>
+  </sect3>
+
+   <sect3>
+    <title>Helm Chart and Container Image Mirroring</title>
+    <para>
+     &productname; uses <link xlink:href="https://www.helm.sh/">Helm</link> as
+     one method to install additional software on the cluster. The logic behind
+     this relies on <literal>Charts</literal>, which are configuration files
+     that tell &kube; how to deploy software and its dependencies
+     and how to configure it. The actual software installed using this method is
```

Comment:
I would rephrase that to `how to deploy and configure the software and its dependencies`.